Cargo.toml

```diff
@@ -3,6 +3,12 @@ name = "rsinit"
 version = "0.1.0"
 edition = "2021"
 license = "GPL-2.0-only"
+description = "minimalistic single binary initramfs init for embedded systems"
+homepage = "https://github.com/michaelolbrich/rsinit"
+repository = "https://github.com/michaelolbrich/rsinit"
+keywords = ["initramfs", "init", "rdinit"]
```

Comment:
What does `rdinit` stand for? I've replaced mentions of it elsewhere with `rsinit` in https://github.com/michaelolbrich/rsinit/pull/3.

Reply:
ramdisk init. The kernel accepts a parameter of that name too:
```
        rdinit=         [KNL]
                        Format: <full_path>
                        Run specified binary instead of /init from the ramdisk,
                        used for early userspace startup. See initrd.
```
which is why I included it as keyword.

Reply:
Ah. Makes sense. Thanks